models/games.models.js

```diff
@@ -67,3 +67,24 @@ exports.selectCommentsByReviewId = (review_id) => {
       });
   });
 };
+
+exports.insertComment = (review_id, body, author) => {
+  return checkExists("users", "username", author).then(() => {
```

Comment:
Good use of a util, but PSQL would also have let you know that the username wasn't valid once you tried to input it. Nothing wrong with this, though!